freqtrade/rpc/api_server/api_schemas.py

```diff
@@ -382,3 +382,7 @@ class BacktestResponse(BaseModel):
 class SysInfo(BaseModel):
     cpu_pct: List[float]
     ram_pct: float
+
+
+class Health(BaseModel):
+    last_process: datetime
```

Comment:
Especially for the API, i think we would want a `last_process_ts` as well (as int - as we do with other endpoints).

otherwise we rely on the client to correctly parse the timestamp - which is usually painful and quite errorprone as i've experienced myself.